### PR TITLE
Added an about page containing all necessary links raised in issue #29

### PR DESCRIPTION
**Pull request by alan-aipe**
Solved issue #29 by adding an links.html file containing all the links stated in the issue. A link icon is provided in the home page of flappy to reach links page.

### CODE REVIEW

**Comment by jig08** [comment]
Hey @alan-aipe , resolve merge conflicts ?

File: links.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+  </style>
+	</head>
+<body>
+<div class="container-fluid">
+	<div class="row">
+		<div class="col-md-4" style="height:100%;background-color:#000">
+		</div>
+		<div class="col-md-4" style="height:100%;background-color:#ffcc7e">
+			<a href="http://fossasia.github.io/flappy-svg/"><img src="back.png" style="position:absolute;width:50px;margin-top:10px;" /></a>
```
Comment (niccokunzmann) [inline links.html]: This must not link to the internet. Please link to the index.html file directly. Otherwise it will not work offline.

**Comment by niccokunzmann** [comment]
I like the design.

**Comment by niccokunzmann** [comment]
Also, please link to the playstore.

**Comment by alan-aipe** [comment]
ok...  I will do the edits. and give another pull request today itself..Thank you for your feedbacks

**Comment by niccokunzmann** [comment]
Looking forward to it :)

**Comment by alan-aipe** [comment]
Plzz review pull request #57 for the new commit..
@jig08 @niccokunzmann @mariobehling

**Comment by jig08** [comment]
@alan-aipe It still has merge conflicts...

**Comment by niccokunzmann** [comment]
#57 obsoletes this.

**Pull request closed without merging**